operator/src/main/java/oracle/kubernetes/operator/ProcessingConstants.java

```diff
@@ -34,4 +34,8 @@
   public static final String DOMAIN_INTROSPECT_VERSION = "weblogic.domainIntrospectVersion";
   public static final String DOMAIN_INPUTS_HASH = "weblogic.domainInputsHash";
   public static final String REMAINING_SERVERS_HEALTH_TO_READ = "serverHealthRead";
+  
+  public static final String KEY = "encodedCredentials";
+  public static final String RESULT = "httpResult";
+  public static final int HTTP_OK = 200;  
```

Comment:
Very small thing ;)... This constant must be many places.  Do we need another? I bet it's available on the HTTP client classes.

Reply:
Fixed, Using HTTP_OK constant from HttpURLConnection class. Thanks.